src/backend/catalog/index.c

```diff
@@ -1007,7 +1007,7 @@ index_create(Relation heapRelation,
 	table_close(pg_class, RowExclusiveLock);
 
 	{							/* MPP-7575: track index creation */
```

Comment:
```suggestion
	if (Gp_role == RP_ROLE_DISPATCH)
	{							/* MPP-7575: track index creation */
```

Since we only track object modification on coordinator, we can prevent going through these codes on segments entirely.

Reply:
Thanks for your comments!  I'll do it.